pom.xml

```diff
@@ -272,6 +296,7 @@
         <felix.url>http://localhost:8080/system/console</felix.url>
         <felix.user>admin</felix.user>
         <felix.password>admin</felix.password>
-        <procyon.version>0.5.36</procyon.version>
+        <jd-core.version>1.1.3</jd-core.version>
+        <cfr.version>0.152-SNAPSHOT</cfr.version>
```

Comment:
why snapshot?

Reply:
Yes sorry about that initially I had the car project checked out locally. Had to experiment to make it decompile a single class